tests/test_models/test_dense_heads/test_freeanchors.py

```diff
@@ -17,19 +17,19 @@ def test_freeanchor(self):
         assert hasattr(mmdet3d.models.dense_heads, 'FreeAnchor3DHead')
         DefaultScope.get_instance('test_freeanchor', scope_name='mmdet3d')
         _setup_seed(0)
-        freeanchor_cfg = _get_detector_cfg(
+        freeanchor_cfg = get_detector_cfg(
```

Comment:
The config file doesn't exist. Please check all config files in the UT of this PR.